flink/pom.xml

```diff
@@ -97,44 +95,44 @@
 
     <dependency>
       <groupId>com.typesafe.akka</groupId>
-      <artifactId>akka-actor_${flink.scala.binary.version}</artifactId>
+      <artifactId>akka-actor_${scala.binary.version}</artifactId>
```

Comment:
Yes, I was referring to `flink.scala.binary.version`, `ignite.scala.binary.version`.
Maybe Ignite doesn't support Scala 2.11? https://github.com/apache/ignite/blob/master/pom.xml#L453